docs/admin_api/user_admin_api.rst

```diff
@@ -51,6 +81,7 @@ This API returns information about a specific user account.
 The api is::
 
     GET /_synapse/admin/v1/whois/<user_id>
+    GET /_synapse/admin/v2/users/<user_id>
```

Comment:
this isn't clear. if the two are altenatives, let's either make that explicit, or (preferably) deprecate one of them and get rid of it here.

Reply:
Isn't it obvious enough that v1 API is superseded by v2?

Reply:
you might be surprised at how easily people seem to get confused by this documentation. No, I don't think it is obvious.